EXOSIMS/Observatory/TwoStarShades_mission.py

```diff
@@ -0,0 +1,567 @@
+from EXOSIMS.Observatory.SotoStarshade import SotoStarshade
+import numpy as np
+import astropy.units as u
+import astropy.constants as const
+import scipy.optimize as optimize
+
+
+class TwoStarShades_mission(SotoStarshade):
+    def __init__(
+        self,
+        scMass=[6000.0, 6000.0],
+        dryMass=[3400.0, 3400.0],
+        counter_1=0,
+        counter_2=0,
+        counter_3=0,
+        counter=0,
+        thrust=450,
```

Comment:
why are you re-defining the thrust input here?

Reply:
resolved